docker/depends/Dockerfile

```diff
@@ -1,20 +1,12 @@
-ARG R_VERSION="4.0.2"
+ARG R_VERSION="4.1"
+ARG FROM_IMAGE="rocker/tidyverse"
 
 # ----------------------------------------------------------------------
 # PECAN FOR MODEL BASE IMAGE
 # ----------------------------------------------------------------------
-FROM rocker/tidyverse:${R_VERSION}
+FROM ${FROM_IMAGE}:${R_VERSION}
 MAINTAINER Rob Kooper <kooper@illinois.edu>
 
-# ----------------------------------------------------------------------
-# UPDATE GIT
-# This is needed for stretch and github actions
-# ----------------------------------------------------------------------
-RUN if [ "$(lsb_release -s -c)" = "stretch" ]; then \
-      echo 'deb http://deb.debian.org/debian stretch-backports main' >> /etc/apt/sources.list \
-      && apt-get update \
-      && apt-get -t stretch-backports upgrade -y git \
-    ; fi
```

Comment:
This deletion is just cleanup -- none of our images have used Stretch since before R 4.0, so this command never runs anymore.

Reply:
Done, thanks!